packages/react/src/components/Tag/Tag.js

```diff
@@ -34,6 +34,8 @@ const Tag = ({
   id,
   type,
   filter,
+  hasCustomIcon,
+  renderIcon: CustomIconElement,
```

Comment:
Would we expect a user to always set their own custom icon? If so, do we just need the `renderIcon` prop, and base the rest of the logic on the presence of this prop?

Reply:
Yes, users are expected to set their own custom icon if they choose to use this `Tag` variation, there's no "default" icon. Users would also need to include the `hasCustomIcon` in their `Tag` component, but yeah, I think needing two variables to activate this behavior may be redundant. 

Would you suggest changing it to only needing `renderIcon`/getting rid of `hasCustomIcon`?

Reply:
Yeah, that's what I was thinking! That way by default there would be no icon and if a user wants to use one they would be expected to provide their own icon. Then we can just check if `renderIcon` exists

Reply:
Got it, pushed the change, thanks!